Sources/DiscordKitCore/Gateway/DecompressionEngine.swift

```diff
@@ -147,3 +148,31 @@ public extension DecompressionEngine {
         return decompressed
     }
 }
+#endif
+
+#if os(Linux) // Consider using this on macOS.
```

Comment:
How does the `SWCompression` compare in terms of performance to Apple's `Compression.framework`? It would only be worth the additional dependancy on macOS if its performance justified it.